libc/config/config.json

```diff
@@ -1,4 +1,10 @@
 {
+  "errno": {
+    "LIBC_CONF_ERRNO_MODE": {
+      "value": "",
```

Comment:
it would probably be best to make the default value `LIBC_ERRNO_MODE_THREAD_LOCAL`

Reply:
I agree but the problem is that we currently don't have any mechanism to distinguish the overlay from the full build mode, and in the overlay mode we want to default to `LIBC_ERRNO_MODE_SYSTEM`.